src/pages/profile-page/components/address-card.tsx

```diff
@@ -18,21 +18,16 @@ import { RiDeleteBin5Fill, RiEditFill } from 'react-icons/ri';
 
 interface AddressCardProps {
   address: Customer.Address;
+  isEdit?: boolean;
 }
 
-export const AddressCard: React.FC<AddressCardProps> = ({ address }) => {
+export const AddressCard: React.FC<AddressCardProps> = ({
+  address,
+  isEdit = false,
+}) => {
   const form = useForm<RegisterFormValues>();
 
-  useEffect(() => {
-    form.reset({
-      [CustomerFieldName.country]: address.country,
-      [CustomerFieldName.city]: address.city,
-      [CustomerFieldName.street]: address.streetName,
-      [CustomerFieldName.postCode]: address.postalCode,
-    });
-  }, [address, form]);
-
-  const [isEditMode, setIsEditMode] = useState<boolean>(false);
+  const [isEditMode, setIsEditMode] = useState<boolean>(isEdit || false);
```

Comment:
```suggestion
  const [isEditMode, setIsEditMode] = useState<boolean>(isEdit);
```
там и так фолс по умолчанию